conda/recipes/ucxx/meta.yaml

```diff
@@ -127,9 +135,15 @@ outputs:
     requirements:
       build:
         - cmake
+      host:
+        {% if cuda_major == "11" %}
+        - cudatoolkit ={{ cuda_version }}
```

Comment:
```suggestion
        - cudatoolkit
```

Reply:
Done in commit ( https://github.com/rapidsai/ucxx/pull/61/commits/44cab6ffa3eb2609a7341036a0cb5da17781f1b3 )